### DISCUSSION TITLE
Support per-repository language timeout

### DISCUSSION THREAD

**Discussion by luxaritas** [lowlighter/metrics#1218] [Ideas]
I've run into an issue where linguist ran for 13 minutes before erroring out for a single repository. It would be great if this situation could be handled so that a repository can be essentially skipped either in the case of an error like this or for super large repositories without preventing other repos from being scanned and without needing to exclude it manually

**Comment by lowlighter** [comment]
Hi 👋 !

This seems like a good idea, I'll add it to the backlog
Thanks a lot for your suggestion !

**Reply by lowlighter** [reply]
[`plugin_languages_analysis_timeout_repositories`](https://github.com/lowlighter/metrics/tree/master/source/plugins/languages#plugin_languages_analysis_timeout_repositories) is currently available on the master/main branches and will be released on latest with next version 👍
